management/server/account.go

```diff
@@ -198,6 +192,27 @@ func BuildManager(
 
 }
 
+// newAccount creates a new Account with a generated ID and generated default setup keys.
+// If ID is already in use we try one more time before returning error
```

Comment:
I think it might make sense to explain why ID can be in use
`If ID is already in use (due to collision) we try one more time before returning error`